dev-manuals/backend/testing.md

```diff
@@ -1,4 +1,162 @@
 # Backend testing
 
 Description of how the backend / different microservices will be tested.  
-Coming next sprint
\ No newline at end of file
+
+## Unit testing
+
+We use [JUnit5](https://junit.org/junit5/) for unit testing,
+[Mockito](https://site.mockito.org/) for mocking and
+[Hamcrest](http://hamcrest.org/JavaHamcrest/) for assertions.
+
+### What to test
+
+We test all classes that contain logic. This means that we do not test classes that only contain getters and setters
+and controllers that only contain a single method that calls a service method.
+
+Typically, we test the `Service` classes of the microservices, but if other classes contain non-trivial logic, we
+also test those, for example if a repository class contains a custom query.
```

Comment:
Imho, this means we should expect a high code coverage for `Service` classes. Should we add this as an expectation to the PR template?

Reply:
Yes I can do this